exercises/concept/high-score/high_score_test.rb

```diff
@@ -0,0 +1,58 @@
+require 'minitest/autorun'
+require_relative '.meta/exemplar'
+
+class HighScoreTest < Minitest::Test
+  def test_blank
+    expected = {}
+    scoreboard = Scoreboard.new
+    assert_equal [], scoreboard.players
```

Comment:
The `players` method is being tested in the very first test case and in the very last test case. In the stub, it's defined at the beginning of the file, yet in the instructions it's the very last task to implement it. This might annoy the students who expect the tasks and tests to be in roughly the same order.

Reply:
I have adapted the exercise/tests to not depend on that anymore :)